cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -61,13 +61,9 @@ public boolean equals(Object o) {
 
     NamespaceConfig that = (NamespaceConfig) o;
 
-    if (!schedulerQueueName.equals(that.schedulerQueueName)) {
-      return false;
-    }
-    if (!principal.equals(that.principal)) {
-      return false;
-    }
-    return keytabPath.equals(that.keytabPath);
+    return schedulerQueueName.equals(that.schedulerQueueName)
```

Comment:
updated. Did not have Guava's equals/hashcode plugin, since I updated intellij to 16.